src/canmatrix/canmatrix.py

```diff
@@ -390,6 +399,7 @@ def phys2raw(self, value=None):
         return raw_value
 
     def raw2phys(self, value, decodeToStr=False):
+        # type: (canmatrix.types.RawValue, bool) -> canmatrix.types.PhysicalValue
```

Comment:
The type checking requires some code rules which might slightly restrict the pythonic freedom. For example here, the `value` variable is RawValue and can't be assigned by `value = self.float_factory(value)` few lines later. Like in statically typed languages, new variable must be used. Now the mypy error is reported here.